boost/boost.bzl

```diff
@@ -222,10 +215,11 @@ def boost_deps():
         ],
     )
 
+    # We're pointing at hedronvision's mirror of google/boringssl:master-with-bazel to get Renovate auto-update. Otherwise, Renovate will keep moving us back to master, which doesn't support Bazel. See https://github.com/renovatebot/renovate/issues/18492
```

Comment:
As I understand it, "live at HEAD" is boringssl's release philosophy, like abseil. There isn't a versioned release scheme we can switch too, but that doesn't mean we don't have good options to reduce PR volume.

I'm imagining the main annoyance here is the PR email/notification volume, since I see you're watching the repo, too. I feel you. I get 'em, too.

I think there are two categories of changes we could make. The first, and the one I'd recommend, is to switch to having the updates just go in silently in as tested, merged branches, rather than PRs (unless they fail CI). That'd automate keeping us using the latest by default while reducing PRs and notifications to a minimum, both for boringssl and the other dependencies that have lower update frequency. Nelson and I had discussed that, but the decision had been to try it this way first to see if was a problem. Switching requires giving Renovate the branch merging permission that github's merge interface has. But if he agrees and is down to give that permission, I'd be delighted to configure Renovate that way. That's how I think we should solve this, if the PRs are sufficiently annoying to be a problem. It's how I configure the repos I own.

[The other option is to still get automated updates as PRs that ping everyone, but to artificially reduce their frequency. I bet there's a way to configure Renovate to only update 2x/month or something. But the other option seems better to me, especially since this is a security-focused dependency, it'd reduce noise further, and commit volume/frequency is generally seen as a positive signal by users.]